src/main/java/seedu/address/logic/parser/ClearCommandParser.java

```
@@ -46,9 +46,9 @@ public ClearCommand parse(String args) throws ParseException {
      * @return ClearMemberCommand or ClearEventCommand
      */
     public ClearCommand matchType(String type) {
-        if (type.equalsIgnoreCase("member")) {
+        if (ParserUtil.isMember(type)) {
             return new ClearMemberCommand();
-        } else if (type.equalsIgnoreCase("event")) {
+        } else if (ParserUtil.isEvent(type)) {
             return new ClearEventCommand();
         } else {
             return null;
```

Instead of returning null here, throw the exception